website/src/pages/index.tsx

```diff
@@ -99,10 +100,10 @@ function Home() {
                     <strong>{plugin.name}</strong>
                   </td>
                   <td style={{ minWidth: 150 }}>
-                    <img
+                    {!plugin.remoteSource && <img
```

Comment:
What's the reason for removing the pub badge on this one? Should still be working, e.g.: https://img.shields.io/pub/v/google_mobile_ads.svg

![sfsf](https://img.shields.io/pub/v/google_mobile_ads.svg)

Reply:
huh, I thought it wasn't working. will update.

Reply:
Thanks!